src/Akeneo/Connectivity/Connection/back/Domain/Marketplace/DTO/GetAllAppsResult.php

```diff
@@ -69,6 +69,19 @@ public function withPimUrlSource(array $queryParameters): self
         );
     }
 
+    public function sortCertifiedFirst(): self
+    {
+        $apps = $this->apps;
+        usort($apps, function ($a, $b) {
+            return $b->isCertified() - $a->isCertified();
```

Comment:
```suggestion
            return $b->isCertified() && !$a->isCertified() ? -1 : 1;
```
Shorter but less obvious
|a|b|result|
|-|-|-|
|0|0|1|
|1|0|1|
|0|1|-1|
|1|1|1|

Reply:
I used your proposition thanks